model-optimizer/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -2,8 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import hashlib
-from xml.etree.ElementTree import Element, SubElement, tostring, ElementTree
 
+from defusedxml import defuse_stdlib
+import defusedxml.ElementTree as ET
```

Comment:
why can't we do:
'from defusedxml.ElementTreeimport Element, ...'
so there will be no need to change code below.

Reply:
There is no Element, SubElement, ElementTree or tostring in defusedxml.ElementTree

Reply:
from ET_defused import Element, ...